SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTest.cpp

```diff
@@ -232,6 +233,90 @@ bool ASpatialFunctionalTest::HasActorDelegation(AActor* Actor, int& WorkerId, bo
 	return bHasDelegation;
 }
 
+void ASpatialFunctionalTest::AddActorInterest(int32 ServerWorkerId, AActor* Actor)
+{
+	ASpatialFunctionalTestFlowController* AuxLocalFlowController = GetLocalFlowController();
+
+	if (AuxLocalFlowController == nullptr || AuxLocalFlowController->WorkerDefinition.Type != ESpatialFunctionalTestWorkerType::Server)
+	{
+		UE_LOG(LogSpatialGDKFunctionalTests, Error, TEXT("Interest changes are only allowed from Server Workers, trying to do it from %s"), *AuxLocalFlowController->GetDisplayName());
+		return;
+	}
+
+	ensureMsgf(ServerWorkerId >= FWorkerDefinition::ALL_WORKERS_ID, TEXT("Invalid ServerWorkerId"));
+	USpatialNetDriver* SpatialNetDriver = Cast<USpatialNetDriver>(GetNetDriver());
+	if (ServerWorkerId < FWorkerDefinition::ALL_WORKERS_ID || SpatialNetDriver == nullptr || Actor == nullptr)
+	{
+		return;
+	}
+	if (ServerWorkerId > NumExpectedServers)
+	{
+		UE_LOG(LogSpatialGDKFunctionalTests, Log, TEXT("Trying to add interest on Server Worker %d and there's only %d; falling back to Server Worker 1"), ServerWorkerId, NumExpectedServers);
+		ServerWorkerId = 1;
+	}
+
+	Worker_EntityId ActorEntityId = SpatialNetDriver->PackageMap->GetEntityIdFromObject(Actor);
+
+	if(ServerWorkerId == FWorkerDefinition::ALL_WORKERS_ID)
+	{
+		for (int i = 0; i != FlowControllers.Num(); ++i)
```

Comment:
Maybe can do `for (ASpatialFunctionalTestFlowController* FlowController : FlowControllers)` ?